eaf-browser.el

```diff
@@ -150,6 +150,14 @@ and will re-open them when calling `eaf-browser-restore-buffers' in the future s
   ""
   :type 'string)
 
+(defcustom eaf-browser-progress-bar-height "5"
+  ""
+  :type 'int)
+
+(defcustom eaf-browser-progress-bar-color "#ff8103"
```

Comment:
Use foreground_color default, if user's theme is same as background, user can customize option, please don't use constant value.